sbin/check-format.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+HERE="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+ROOT=$HERE/..
+READIES=$ROOT/deps/readies
+. $READIES/shibumi/functions
```

Comment:
why do we need this?

Reply:
Yes. It's actually `. $READIES/shibumi/defs`
It "includes" all relevant automation code.